source/mainnet/smart-contracts/guides/compile-module.rst

```diff
@@ -38,19 +38,6 @@ In order to build a smart contract, run:
 
 This uses Cargo_ for building, but runs further optimizations on the result.
 
-.. Note::
-
-   If you receive an error similiar to the one below, you need to :ref:`install an earlier version of the rustup compiler<rust-compiler-issue>`.
-
```

Comment:
Removing this is fine, but I am wondering whether it would make sense for now to keep a revised notice

```
If you receive an error similiar to the one below, you need to update your ``cargo-concordium`` to version 3.0 or later.
```

Reply:
This is because some might still have an older version of cargo-concordium installed (which is fine since contracts produced by that version are still going to be valid).